schema.graphql

```diff
@@ -89,6 +89,15 @@ type Reward @entity {
   id: ID!
 	asset: Bytes!
 	committed: BigInt!
+  token: Token!
+}
+
+type Token @entity {
+  id: ID!
+  label: String!
+  address: Bytes!
+  decimal: Int!
+  iconHash: String!
```

Comment:
Probably worth it to add a `@derivedFrom` decorator here since we'll be putting tokens via grant and you won't have to keep track of the tokens array in workspace `https://thegraph.com/docs/en/developer/create-subgraph-hosted/#reverse-lookups`

Reply:
But we want to show the list of tokens accepted by the workspace in the dropdown menu. For that, we might need to keep track of all the tokens created by the workspace.

Reply:
Yeah that remains the same, it's just an easier way to store the list -- should def visit the link I sent